src/modules/mavlink/mavlink_main.h

```diff
@@ -627,10 +627,11 @@ class Mavlink : public ModuleParams
 	bool			_broadcast_address_found{false};
 	bool			_broadcast_address_not_found_warned{false};
 	bool			_broadcast_failed_warned{false};
-	uint8_t			_network_buf[MAVLINK_MAX_PACKET_LEN] {};
-	unsigned		_network_buf_len{0};
 #endif
 
+	uint8_t			_buf[MAVLINK_MAX_PACKET_LEN] {};
+	unsigned		_buf_len{0};
```

Comment:
I find this variable name confusing. Usually a buffer length would indicate the max buffer size but in this case it could be more like a `buf_len_to_write`.

Reply:
Renamed `_buf_len -> _buf_fill`, but I'm not picky if you have a better suggestion.